sources/AppBundle/Controller/Admin/Membership/MemberTypeEnum.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppBundle\Controller\Admin\Membership;
+
+enum MemberTypeEnum: int
+{
+    case MEMBER_PHYSICAL = 0;
+    case MEMBER_COMPAGNY = 1;
+    case ALL_MEMBERS = 2;
```

Comment:
```suggestion
    case MemberPhysical = 0;
    case MemberCompany = 1;
    case AllMembers = 2;
```

Le standard pour les enums c'est plutôt du `PascalCase` : https://www.php-fig.org/per/coding-style/#9-enumerations

Vu qu'on se base sur `PER-CS`, pour le moment en v2, on aura le soucis en v3 donc autant anticiper.

Et `Compagny` c'est pas correct, il faut écrire `Company`. Un des soucis à faire du code en anglais 😛